core/settings/qPrefUnit.h

```diff
@@ -77,7 +78,8 @@ public slots:
 	void show_units_tableChanged(bool value);
 	void temperatureChanged(int value);
 	void temperatureStringChanged(const QString& value);
-	void unit_systemChanged(const QString& value);
+	void unit_systemChanged(int value);
```

Comment:
it seems like we don't reference this `int` based version elsewhere...

Reply:
no not in this PR, but it will come in the next PR.